packages/validator/src/services/prepareBeaconProposer.ts

```diff
@@ -1,9 +1,10 @@
 import {Epoch, bellatrix} from "@lodestar/types";
-import {Api, ApiError, routes} from "@lodestar/api";
-import {BeaconConfig} from "@lodestar/config";
+import {Api, ApiError} from "@lodestar/api";
 import {SLOTS_PER_EPOCH} from "@lodestar/params";
+import {BeaconConfig} from "@lodestar/config";
 
-import {IClock, LoggerVc, batchItems} from "../util/index.js";
+import {ProposerPreparationData} from "@lodestar/api";
```

Comment:
`ProposerPreparationData` should probably be imported with the other `from "@lodestar/api";` imports on line 2

Reply:
Thanks my `yarn install` is broken because of https://github.com/ChainSafe/lodestar/issues/4767 so I think my ts auto-imports are messed up

Reply:
It's not properly exported https://github.com/ChainSafe/lodestar/actions/runs/4245050914/jobs/7380036254#step:7:32 we should create an issue for this

Reply:
Not sure if the extra effort would be worth it and if it is done in this PR someone looking at why this was exported will see it here. But would like to here one of the maintainers opinion on this.